### PR TITLE
Keybindings

### PR DESCRIPTION
**Pull request by joonalillfors**

### CODE REVIEW

File: client/src/components/onboarding/onboarding.scss [hunk 1]
```diff
@@ -91,6 +91,13 @@ $padding: $spacer * 2;
   padding: 0 $padding;
 }
 
+.full-modal {
```
Comment (HenrikHelenius) [inline client/src/components/onboarding/onboarding.scss]: what is this

File: client/src/components/onboarding/onboarding.js [hunk 2]
```diff
@@ -196,57 +197,78 @@ class Onboarding extends React.Component {
...
+        break
+      case 'Escape': this.handleCloseIntro()
+        break
+      default: break
+    }
+  }
+
   render() {
     const { step } = this.state
+    let myEl = null
```
Comment (HenrikHelenius) [inline client/src/components/onboarding/onboarding.js]: rename, element

File: client/src/components/onboarding/onboarding.js [hunk 3]
```diff
@@ -196,57 +197,78 @@ class Onboarding extends React.Component {
...
+              <InlineSVG src={ArrowIcon} />
+            </button>
+          </div>
           <button
             type="button"
+            className={step !== final ? css('hidden') : css('done-button')}
             onClick={this.handleCloseIntro}
-            className={step === final ? css('skip-button', 'hidden') : css('skip-button')}
-          >skip
+          >Close tutorial
```
Comment (HenrikHelenius) [inline client/src/components/onboarding/onboarding.js]: newline

**Review by HenrikHelenius — CHANGES_REQUESTED** [review]

File: client/src/components/onboarding/onboarding.scss (hunk 1 above)
Comment (joonalillfors) [reply client/src/components/onboarding/onboarding.scss]: the extra div the eventhandler is listening to

Comment (joonalillfors) [reply client/src/components/onboarding/onboarding.scss]: changed the classname

**Review by HenrikHelenius — APPROVED** [review]

**Comment by joonalillfors** [comment]
[[task] [misc] Enter + ctrl submits form Joona](https://trello.com/c/bL5VzwTW/149-task-misc-enter-ctrl-submits-form-joona)

**Pull request merged**